### DISCUSSION TITLE
Dropdown: Force Selection

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#189] [PrimeVue]
I wonder if there is a way to keep a value that doesn't exist in options just like AutoComplete Force Selection property being disabled, I have a scenario where I have a paginated dropdown, each time the user moves to the next pagination, I refetch the server to get the next 6 options. Also, I don't want to use the editable property since I don't want the user to enter values manually.

**Comment by tugcekucukoglu** [comment]
AutoComplete provides real-time suggestions as the user types, allowing them to choose from the suggestions or continue typing but Dropdown expands to reveal the available options, allowing the user to select one of them. In general, in terms of user experience, forcing a selection with a drop-down list is disapproved. But some emits could be used that will be called when the dropdown list is closed or the user clicks out. For example: `hide` or `blur`. For the other emits: https://primevue.org/dropdown/#api.dropdown.emits

**Comment by ghost** [comment]
> AutoComplete provides real-time suggestions as the user types, allowing them to choose from the suggestions or continue typing but Dropdown expands to reveal the available options, allowing the user to select one of them. In general, in terms of user experience, forcing a selection with a drop-down list is disapproved. But some emits could be used that will be called when the dropdown list is closed or the user clicks out. For example: `hide` or `blur`. For the other emits: https://primevue.org/dropdown/#api.dropdown.emits

So just to simplify, I would like to have the behavior of the editable property in the dropdown where it accepts values that aren't in options, without it actually being editable. So just the same behavior except that I don't want the user to be able to type values manually and only select them from displayed options.

**Comment by cagataycivici** [comment]
There is no such option in Dropdown at the moment but I think it makes sense to implement it.
